syserol/model.py

```diff
@@ -52,3 +56,42 @@ def cross_validation():
         matrix[test_index, 6:13] = pred_matrix[test_index, len(glycan) : len(glycan) + 6]  # store predicted values
 
     return matrix
+
+def class_predictions():
```

Comment:
Just take `subjects_matrix` as an argument to this function. A function should do one, focused thing.

Reply:
This function also needs a more descriptive name.